cmd/kubectl-directpv/discover.go

```diff
@@ -218,7 +218,7 @@ func discoverMain(ctx context.Context) {
 				eprintf(true, "discovery failed; %v\n", err)
 				os.Exit(1)
 			case <-ctx.Done():
-				eprintf(true, ctx.Err().Error())
+				eprintf(true, "%v", ctx.Err())
```

Comment:
why are we changing this?

Reply:
eprintf() requires `bool, format, data` I think that's why  and the new Go vet complains for incorrect usage.